course6/toyapp_alexs/displaylib/transformation3d.cc

```diff
@@ -0,0 +1,75 @@
+#include <cmath>
```

Comment:
Order you headers from the mosty specific. local ones, to the ost generic ones.

So always put the std headers at the end.

Reply:
ok, we usually do it vice versa.